tools/ci/before_install.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -v
 
+sudo sed -i 's/127.0.1.1/127.0.0.1/' /etc/hosts
+git config --global user.name "ManageIQ"
+git config --global user.email "contact@manageiq.org"
```

Comment:
@odidev What prompted adding these 2 lines?  Mostly curious why this wasn't needed on amd64

Reply:
I was getting the following error in travis :
```
fatal: ambiguous argument 'HEAD~': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
F.
*** Please tell me who you are.
Run
  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
to set your account's default identity.
Omit --global to set the identity only in this repository.
```

While I was trying to build this package for arm64 and amd64(except xenail). So I have added these line and now the test cases are passing.

Reply:
this part of the PR is definitely odd.  We need to resolve this before merging.